### DISCUSSION TITLE
Other models for RIS in Sionna

### DISCUSSION THREAD

**Discussion by HelmutWolfgang** [NVlabs/sionna#686] [General]
Hello together,

I have a question regarding the implementation of RIS in Sionna ray tracing. Currently, only one way to model a RIS in a scene is implemented. However, different approaches to model RIS in a communication link exist, such as using the bistatic radar equation with the RIS RCS defined in a look-up table generated from full-wave simulation or other analytical approaches.

Is the implementation of other approaches on the agenda for further updates and if not, how would you tackle a task like this?

Thank you.

**Comment by jhoydis** [comment]
Hi @HelmutWolfgang,

A very important aspect of the RIS model in Sionna is that we can use it for both, the computation of channel impulse responses as well as coverage maps (ray-based model). I am not aware of any other model that would allow us to do this. You are of course welcome to implement any other model you like, but we do not have this on our agenda for now.

**Reply by potatoess11** [reply]
what about RCS of the  single point or multipoint modeling or calculation methods in ISAC channels in sionna
